Dangerfile

```diff
@@ -0,0 +1,9 @@
+jira.check(
+  key: ["CSDK", "CF", "SDKONCALL"],
+  url: "https://revenuecats.atlassian.net/browse",
+  search_title: true,
+  search_commits: false,
+  fail_on_warning: false,
+  report_missing: true,
+  skippable: true # skippable by adding [no-jira] to PR title or body
```

Comment:
🧡 like the skippable, but maybe also enforce a skip reason? Like if you use the [no-jira] we should have a description that makes it obvious why we skipped? like require `skip jira reason:`
Thoughts?

Reply:
I like it. The thing is that this is a plugin and it doesn't have the option. I would also like to change the warning message. Maybe I fork it?

Reply:
Yeah, it looks abandoned 😿 I think your best bet is a fork. I looked at the other forks, and nothing sticks out as one we should use.

Reply:
Yes... the goal of this sprint is to work on release trains so I might merge it like this and work on the fork in the future. We are going to still use Danger for the release trains (to check if PRs are label correctly) so this PR is still useful